tlsoverridefirefox/filter.go

```diff
@@ -0,0 +1,33 @@
+package tlsoverridefirefox
+
+import (
+	"strings"
+)
+
+// FilterOverrides accepts as input the contents of a Firefox cert_override.txt
+// and a host suffix to blacklist (usually "bit").  It returns the contents of
+// a new Firefox cert_override.txt that contains all overrides as the input
+// file except for those that match the host suffix.
+func FilterOverrides(overrides, blacklistedHostSuffix string) (string, error) {
+	result := ""
+
+	overridesSlice := strings.Split(overrides, "\n")
+
+	for _, override := range overridesSlice {
+		tabSplit := strings.Split(override, "\t")
+		hostAndPort := tabSplit[0]
+
+		colonSplit := strings.Split(hostAndPort, ":")
+		host := colonSplit[0]
```

Comment:
change to use net.SplitHostPort (handles IPv6 addresses properly)

Reply:
Good catch.  I wasn't expecting IPv6 syntax to matter here since public CA's don't issue TLS certs for IP addresses, but now I realize that of course it's entirely plausible for a Firefox user to configure a certificate override for an IP address even though the cert isn't issued to the IP.

Reply:
Fixed.